### PR TITLE
[CNVS Upgrade] Remove inverse classes

### PR DESCRIPTION
**Pull request by jfurrow**
This PR removes the `inverse` classes throughout the application.

### CODE REVIEW

File: src/js/components/FilterByTaskState.js [hunk 1]
```diff
@@ -17,7 +17,7 @@ class FilterByTaskState extends React.Component {
 
   getItemHtml(item) {
     return (
-      <span className="inverse">
+      <span>
```
Comment (mlunoe) [inline src/js/components/FilterByTaskState.js]: Let's remove the extra span here

Comment (jfurrow) [inline src/js/components/FilterByTaskState.js]: Good call, done!

**Comment by mlunoe** [comment]
You'll find `inverse` in AlertPanel, and `container-pod-divider-inverse` too

**Comment by mlunoe** [comment]
Let's ⛵️

**Pull request merged**